ui/app/serializers/deployment.js

```diff
@@ -8,18 +8,25 @@ export default ApplicationSerializer.extend({
   },
 
   normalize(typeHash, hash) {
-    hash.TaskGroupSummaries = Object.keys(get(hash, 'TaskGroups') || {}).map(key => {
-      const deploymentStats = get(hash, `TaskGroups.${key}`);
-      return assign({ Name: key }, deploymentStats);
-    });
+    if (hash) {
```

Comment:
When is `hash` falsy? I thought it was always provided to `normalize`.

Reply:
If a job has no deployments, the response for latest deployment is a 200 with `null` for the body.